### DISCUSSION TITLE
Is there a way to add `Tooltip` on `MenuItem`?

### DISCUSSION THREAD

**Discussion by goldwind-ting** [ant-design/ant-design#38756] [Q&A] [answered]
Is there a way to add `Tooltip` on `MenuItem`?
for example, show the tips when mouse hovers the `Introduction`
![image](https://user-images.githubusercontent.com/63939636/202892618-edb4574b-d24a-4917-9018-ee88be1e3e7e.png)

**Comment by natashaamin** [comment, marked as answer]
Can we see the code? Normally its possible

**Reply by goldwind-ting** [reply]
> Can we see the code? Normally its possible
sorry, my bad, it works now.
